projects/packages/forms/src/contact-form/css/grunion.css

```diff
@@ -235,6 +235,10 @@
 	box-sizing: border-box;
 }
 
+:where( .wp-block-jetpack-contact-form .wp-block-separator.is-style-default ) {
+	max-width: var(--wp-preset--spacing--80, 150px );
```

Comment:
I expanded about `is-style-default` [here](https://github.com/Automattic/jetpack/pull/40967#issuecomment-2595873353), so If you wanted an override for the default style, you should just target the block when inside the forms block.

Separator block has this [width style](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/separator/theme.scss#L15) in `theme.scss`. Which means that these styles are only loaded if the [active theme opts in for them](https://github.com/WordPress/gutenberg/blob/trunk/docs/how-to-guides/themes/theme-support.md#opinionated-block-styles). That explains why you can reproduce in some themes and not in others.

Reply:
A possible problem would be though if a theme has a similar width style for `Separator` block and that would create a mismatch for this block in and out of Forms block. It's more design related discussion and designers/themers should chime in.

Reply:
Thanks for the suggestion. I removed the styles for the default separator class and it worked!